src/v/redpanda/application.cc

```diff
@@ -147,6 +160,11 @@ void application::validate_arguments(const po::variables_map& cfg) {
     if (!cfg.count("redpanda-cfg")) {
         throw std::invalid_argument("Missing redpanda-cfg flag");
     }
+    if (
+      config::shard_local_cfg().enable_proxy()
+      && !cfg.count("pandaproxy-cfg")) {
```

Comment:
what is `pandaproxy-cfg`? I think this would be good to add to the cover letter examples, unless I missed it? do we only need one of these in that `pandaproxy-cfg` would imply enabling proxy?

Reply:
It's code that should not have survived.